### ISSUE TITLE
Issues with next version of ggplot2

### ISSUE THREAD

**Issue by thomasp85** [Biometris/statgenGWAS#10]
Hi

We are preparing the next release of ggplot2 and our reverse dependency checks show that your package is failing with the new version. I believe the issue is in this line in your test suite:

https://github.com/Biometris/statgenGWAS/blob/8cab202b107bbabad25633d3792d947ff8fe76f5/inst/tinytest/test_plots.R#L121-L122

where ggplot2 now throws a deprecation warning and you expect the plotting to be silent, but I may be wrong

You can install the release candidate of ggplot2 using `devtools::install_github('tidyverse/ggplot2@v3.4.0-rc')` to test this out.

We plan to submit ggplot2 by the end of October and hope you can have a fix ready before then

Kind regards
Thomas

**Comment by BartJanvanRossum** [comment]
Hi Thomas,

I just checked, but instead of a warning I get an error:
```r
Error in geom_line() : Problem while converting geom to grob.
ℹ Error occurred in the 2nd layer.
Caused by error in `vec_size()`:
! `x` must be a vector, not an environment.
```

The cause of the problem seems to be in geom_line(). This very basic code is enough for me to replicate it:
```r
library(ggplot2)
dat <- data.frame(x = 1:3, y = 1:3)
ggplot(aes(x = x, y = y), data = dat) +
  geom_line()
```
Could it be that I need the develop version of some other package as well to get this working?

**Comment by thomasp85** [comment]
In that case I think the issue is with the development version of vctrs - I'll check and get back to you

**Comment by thomasp85** [comment]
Can you provide a traceback for the error?

**Comment by thomasp85** [comment]
thanks - can I get the output of `sessioninfo::session_info()` as well

**Comment by BartJanvanRossum** [comment]
That does fix the error message and now raises the deprecation warning you mentioned in the original post. I will make sure to fix that.

**Comment by thomasp85** [comment]
Great - you can assume vctrs will be released before ggplot2

**Comment by BartJanvanRossum** [comment]
Fixed in version 1.0.9
